pkg/apis/apps/v1beta1/register.go

```diff
@@ -35,7 +37,7 @@ func Resource(resource string) schema.GroupResource {
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes, addDefaultingFuncs, addConversionFuncs)
-	AddToScheme   = SchemeBuilder.AddToScheme
+	AddToScheme   = announced.CombineSchemeBuilders(SchemeBuilder.AddToScheme, v1.AddToScheme).AddToScheme
```

Comment:
With https://github.com/kubernetes/kubernetes/pull/41788? Why should we die on that? Only on different types for same gvk.